newdle/client/src/components/creation/timeslots/Timeline.js

```diff
@@ -211,13 +213,22 @@ function TimelineInput({minHour, maxHour}) {
           <div className={styles['candidates-group']} key={i}>
             {rowCandidates.map(time => {
               const slotProps = getCandidateSlotProps(time, duration, minHour, maxHour);
+              const participants = availability?.find(a => a.startDt === `${date}T${time}`);
               return (
                 <CandidateSlot
                   {...slotProps}
                   key={time}
                   isValidTime={time => !candidates.includes(time)}
                   onDelete={() => handleRemoveSlot(time)}
                   onChangeSlotTime={newStartTime => handleUpdateSlot(time, newStartTime)}
+                  text={
+                    participants &&
+                    plural(participants.availableCount, {
+                      zero: 'No participants registered',
```

Comment:
English doesn't have a special plural form for `zero`, you need `_0` here.

https://js-lingui-git-next.lingui-js.now.sh/tutorials/react.html#beware-of-zeroes

Reply:
Too bad - probably needs to be separated with a ternary operator then because `zero` probably won't work...  :/

I'd also open an issue in the lingui repo about this.

Reply:
lingui/js-lingui:770

Reply:
lingui/js-lingui#770

Reply:
Since it was just a bug in the typescript definitions in the end, let's use `0: ...` here